client/index.js

```diff
@@ -68,7 +83,7 @@ var newConnection = function() {
 	}));
 
 	sock.onclose = function() {
-		console.error("[WDS] Disconnected!");
+		log("error", "[WDS] Disconnected!");
```

Comment:
Good point. We could exclude the compilation errors from the log level, so far I think no-one has complained about that.
I'll update this PR later today.